networking/custom-domain.html.markerb

```diff
@@ -58,37 +91,90 @@ fly certs add example.com
   Status                      =
 ```
 
-Running `fly certs add` starts the process of getting a certificate. 
-
-Run `fly certs show` to get the details needed for your next step. For example:
+If you need a wildcard domain, put quotes around the hostname to avoid shell expansion:
 
 ```cmd
-fly certs show example.com
-```
-```output
-  Hostname                    = example.com
-  Configured                  = true
-  Issued                      = ecdsa, rsa
-  Certificate Authority       = lets_encrypt
-  DNS Provider                = enom
-  DNS Validation Instructions = CNAME _acme-challenge.example.com => example.com.5xzw.flydns.net.
-  DNS Validation Hostname     = _acme-challenge.example.com
-  DNS Validation Target       = example.com.5xzw.flydns.net
-  Source                      = fly
-  Created At                  = 1m24s ago
-  Status                      = Ready
+fly certs add "*.example.com"
 ```
 
-The **DNS Validation Instructions** are an optional next step. For a short time (minutes) after we start the process of generating
-the first-ever certificate for your site, trying to load that site with an HTTPS URL will generate errors. If you'd like to make sure
-those errors aren't ever visible, you can use a DNS challenge to pre-generate the certificate. 
+Running `fly certs add` starts the process of getting a certificate. `Configured` should be true and `Status` will change to `Ready` when the certificates are available. You can run `fly certs show <hostname>` to check the status.
+
+
+### (Optional) Validate with an ACME challenge
+
+Validate your domain with an ACME challenge if one or more of the following scenarios apply:
+
+- You want to issue a certificate before creating the CNAME or A/AAAA records to point to your app.
+- You're using a proxy like Cloudflare, which prevents our systems from verifying the source IP addresses.
```

Comment:
this is embarassing, but off the top of my head I don't know how to link to other anchors within the same document on our docs! If this is supported, I think it would be good to directly link to the Universal SSL suggestion here.

Reply:
You can click on the little `#` icon that appears when hovering over a doc header! So maybe something like this? (This is with the preview link.)

```suggestion
- You're using a proxy like [Cloudflare](https://aa-docs-2.fly.dev/docs/networking/custom-domain/#i-use-cloudflare-and-there-seems-to-be-a-problem-issuing-or-validating-my-fly-io-tls-certificate), which prevents our systems from verifying the source IP addresses.
```

Reply:
ha no worries, you can do it with just the `#` and the part after it. 😄 

good idea!